queries.sql

```diff
@@ -1,3 +1,15 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT * FROM animals WHERE neutered = true AND escape_attempts<3;
```

Comment:
- [ ] Again here the query should list the name and not the entire record, kindly fix this.
![image](https://user-images.githubusercontent.com/81932478/212674567-c995e8ea-cafb-44df-aa2c-ace90530179e.png)